src/test/ui/suggestions/issue-85945-check-where-clause-before-suggesting-unsized.stderr

```diff
@@ -0,0 +1,13 @@
+error[E0277]: the size for values of type `str` cannot be known at compilation time
+  --> $DIR/issue-85945-check-where-clause-before-suggesting-unsized.rs:4:16
+   |
+LL | fn foo<T>(_: &T) where T: Sized {}
+   |        - required by this bound in `foo`
```

Comment:
Ideally this should point at the `T: Sized` span instead of where it was introduced, but it's ok either way.